examples/evm/squaring/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@examples/squaring-evm",
+  "private": true,
+  "scripts": {
+    "build": "forge build",
+    "test": "vitest run --reporter verbose",
+    "test:sol": "forge test"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^5.4.0",
+    "@openzeppelin/contracts-upgradeable": "^5.4.0",
+    "@openzeppelin/foundry-upgrades": "^0.4.0",
+    "@satlayer/contracts": "workspace:*",
+    "@satlayer/testcontainers": "workspace:*",
```

```suggestion
    "@satlayer/contracts": "workspace:*",
```

Move these two to devDep